src/pages/Swap/SwapBox/SwapContext.ts

```diff
@@ -0,0 +1,14 @@
+import { RoutablePlatform } from '@swapr/sdk'
+
+import { createContext, Dispatch, SetStateAction } from 'react'
+
+import { UseDerivedSwapInfoResult } from '../../../state/swap/hooks'
+
+export type PlataformOverride = RoutablePlatform | null
```

Comment:
```suggestion
export type PlatformOverride = RoutablePlatform | null
```
change in all imports. :)